ixdiagnose/utils/command.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+
+from typing import Optional, Union
+
+from ixdiagnose.exceptions import CallError
+
+from .run import run
+
+
+class Command:
+
+    def __init__(
+        self, command: Union[str, list], description: str, shell: bool = False,
+        serializeable: bool = True, safe_returncodes: list = None, env: Optional[dict] = None,
```

Comment:
The correct spelling is `serializable` (you use both in this project)

Reply:
Also why not set it to `False` by default? Very few commands return JSON actually.